src/windows/post_event.c

```diff
@@ -65,6 +66,12 @@ static UINT keymask_lookup[8] = {
 	VK_RMENU
 };
 
+// See https://stackoverflow.com/a/4555214 and its comments
+int to_windows_coordinates(int coordinate, int screen_size){
```

Comment:
let's make this function static inline and call it something like convert_to_relitive_position.